mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
```

Comment:
A lot of these configs look useful. Did you have a look at these? If it spams to much we shouldn't use it https://mypy.readthedocs.io/en/stable/command_line.html#configuring-warnings

Reply:
I think we can increase the warning thresholds in subsequent PRs. I think many of them need tiny code changes and I wanted to keep the very first setup basic / non-invasive.